bin/mod/ent-ecr

```diff
@@ -189,16 +189,20 @@ ecr-generate-custom-resource() {
 }
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 ent-ecr-install() {
-  local VERSION_TO_INSTALL CONFLICT_STRATEGY
+  local VERSION_TO_INSTALL CONFLICT_STRATEGY TENANT_CODE
```

Comment:
@alepintus can you rename this as `TENANT_NAME` and all its references please, so we have everything aligned with the official documentation and all the templates?

Reply:
Hi @pietrangelo, 
Thanks for the review.
The TENANT_CODE is an internal variable not visible to the users,  but I changed the help message in the last commit.